eis_toolkit/training_data_tools/class_balancing.py

```diff
@@ -11,24 +11,27 @@
 def balance_SMOTETomek(
     X: Union[pd.DataFrame, np.ndarray],
     y: Union[pd.Series, np.ndarray],
-    sampling_strategy: Union[float, str, dict] = "auto",
+    sampling_strategy: Union[float, Literal["minority", "not minority", "not majority", "all", "auto"], dict] = "auto",
     random_state: Optional[int] = None,
 ) -> tuple[Union[pd.DataFrame, np.ndarray], Union[pd.Series, np.ndarray]]:
```

Comment:
Just realized now that this is a more accurate way to type the return values
```suggestion
) -> Union[tuple[pd.DataFrame, pd.Series]], tuple[np.ndarray, np.ndarray]]:
```